packages/client/src/App.sass

```diff
@@ -432,3 +435,22 @@ hr
   border-radius: 4px
 .table-border > :not(:last-child)
   border-bottom: 1px solid $grey-light
+.confirmation
+  white-space: nowrap
+  padding: 15px 14px
+  font-size: 0.875rem
+  box-shadow: none
+  border: 1px solid $black
+  border-radius: 8px
+  margin-right: 8px
+  svg
+    margin-right: 12px
+.confirmations
+  overflow: auto
+svg
```

Comment:
Let's have this part of svg styling nested under the confirmation class so it doesn't pollute the global style